src/mantarray_file_manager/file_writer.py

```diff
@@ -25,9 +29,14 @@
 from .constants import TRIMMED_TIME_FROM_ORIGINAL_END_UUID
 from .constants import TRIMMED_TIME_FROM_ORIGINAL_START_UUID
 from .constants import UTC_TIMESTAMP_OF_FILE_VERSION_MIGRATION_UUID
+from .exceptions import MantarrayFileNotLatestVersionError
+from .exceptions import TooTrimmedError
+from .exceptions import UnsupportedArgumentError
 from .exceptions import UnsupportedFileMigrationPath
+from .files import _find_start_index
```

Comment:
note for future: in general, if you need to import a function, then it's not really private, so it shouldn't have the leading underscore

Reply:
If a function is used across modules within a library, but wouldn't be of general use to someone using the library itself, then you can just not include it in `__init__.py`